app/controllers/api/v1/keywords_controller.rb

```diff
@@ -32,6 +34,21 @@ def create_success_response
           meta: I18n.t('csv.upload_success')
         }
       end
+
```

Comment:
I think we should include private here
```suggestion
private
```

Reply:
Private is already added at line 26